packages/remix-dev/config.ts

```diff
@@ -371,22 +371,20 @@ export async function readConfig(
   let routes: RouteManifest = {
     root: { path: "", id: "root", file: rootRouteFile },
   };
+  let addRoutes = (routes: RouteManifest) =>
+    Object.values(routes).forEach((route) => {
+      routes[route.id] = { ...route, parentId: route.parentId || "root" };
+    });
```

Comment:
@ftonato These changes are causing tests to fail, so let's revert them (for now)

```suggestion
  function addRoutes(data: RouteManifest) {
    for (let key of Object.keys(data)) {
      let route = data[key];
      routes[route.id] = { ...route, parentId: route.parentId || "root" };
    }
  }
```